### PR TITLE
Update Burnt.podspec

### PR DESCRIPTION
**Pull request by samora**
SPAlert is currently at version 5, causing the CI builds to fail. `SPAlertIconPreset` among other types are no longer available in version 5.

If you approve this can you kindly push a new version to NPM? Thanks 🙏

### CODE REVIEW

**Comment by samora** [comment]
@nandorojo

**Comment by nandorojo** [comment]
Hey I did this with 0.12 an hour ago

**Comment by nandorojo** [comment]
Let me try again if it didn't work

**Comment by nandorojo** [comment]
See https://github.com/nandorojo/burnt/releases/tag/v0.12.1 and https://github.com/nandorojo/burnt/releases/tag/v0.12.0

**Comment by samora** [comment]
Thanks @nandorojo! And thanks for the great work on Solito, Zeego and other related libraries. Allows us to to do some great work all the way here in Accra, Ghana. Stay blessed 🙏
